.vib/opencart/cypress/cypress/fixtures/customers.json

```diff
@@ -1,8 +1,9 @@
 {
-  "newGuest": {
+  "newCustomer": {
     "firstName": "Abigail",
     "lastName": "Somersat",
     "email": "abby@example.com",
+    "password": "P4ssW0rd",
     "phone": "653111345",
```

Comment:
This param is no longer used, you can remove it.

Reply:
It is used in `cypress/integration/opencart_spec.js`.